### PR TITLE
Add LexDAO & ANJmol space folders

### PR DESCRIPTION
**Pull request by Ro5s**
This PR reflects the new space format (as recommended in https://github.com/balancer-labs/snapshot/pull/48). Thanks!

### CODE REVIEW

File: spaces/anjmol/skin.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.name {
```
Comment (bonustrack) [inline spaces/anjmol/skin.scss:1]: This should be changed to the space id
```suggestion
.anjmol {
```

File: spaces/lexdao/skin.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.name {
```
Comment (bonustrack) [inline spaces/lexdao/skin.scss:1]:
```suggestion
.lexdao {
```

**Review by bonustrack — APPROVED** [review]

**Review by bonustrack — CHANGES_REQUESTED** [review]
can you do the changes i've suggested?

File: spaces/lexdao/skin.scss (hunk 2 above)
Comment (Ro5s) [reply spaces/lexdao/skin.scss:1]: many thanks!!

**Comment by Ro5s** [comment]
I just made the requested changes @bonustrack // appreciate the notes! :smile_cat:

**Comment by bonustrack** [comment]
@Ro5s If the skins are same as the example (the default skin) then you can remove these files in the PR.

**Comment by Ro5s** [comment]
I took the opportunity to refresh and add some custom colors to the skin files for LexDAO + ANJmol. Thanks for checking status @bonustrack !

**Comment by bonustrack** [comment]
@Ro5s This PR need to be updated with base branch, now it's out-of-date.

**Comment by Ro5s** [comment]
Done! @bonustrack thanks :~)

**Review by bonustrack — APPROVED** [review]

**Pull request merged**